addons/circulation/stringtable.xml

```diff
@@ -312,6 +312,14 @@
             <Spanish>Utilice el desfibrilador AED</Spanish>
             <Chinese>使用AED除顫器</Chinese>
 			<Czech>Použít AED defibrilátor</Czech>
+        </Key>
+		<Key ID="STR_KAT_circulation_DefibS_Action_Use">
+            <English>Use AED Station</English>
+            <German>Benutze AED Station</German>
+            <Polish>Użyj stacja AED</Polish>
+            <Spanish>Utilice el estación AED</Spanish>
+            <Chinese>使用AED除顫器</Chinese>
+			<Czech>Použít AED stanice</Czech>
```

Comment:
this should be <Czech>Použít AED stanici</Czech>

Reply:
Is in the new commit